deploy/onpublish.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Travis build trigger should have environment variables COMMIT_USER, COMMIT_EMAIL and COMMIT_TOKEN setup.
+
+SOURCEFILENAME=".deploy/ddsclient-generic-gcb01.spec"
+TARGETFILENAME="ddsclient-$TRAVIS_TAG-gcb01.spec"
+COMMITMSG="DukeDSClient module update $TRAVIS_TAG"
+COMMITBRANCH="master"
+
+# Encode the spec file as base64
+BASE64CONTENT=`base64 $SOURCEFILENAME`
+
+# Upload file to helmod repo
+curl -i -X PUT -H "Authorization: token $COMMIT_TOKEN" \
```

Comment:
I see this is using https://developer.github.com/v3/repos/contents/#create-a-file, which only works for new files. I assume this will be the case most of the time, since the file name depends on the tag/version, but are there any cases where the file will already exist?